src/ui/screens/login_screen.py

```
@@ -32,9 +32,9 @@ def __init__(self, screen, api: ApiClient):
         self.show_debug_button = True
 
         # state
-        self.account = "test1"
+        self.account = "testuser123"
         self.email = ""
-        self.password = "123"
+        self.password = "testpass123"
```

Ruff (0.14.6)

Possible hardcoded password assigned to: "password"

(S105)